src/components/Base.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta, ColorPalette, ColorItem, Typeset } from '@storybook/addon-docs/blocks';
+import theme from '../app/theme';
+
+<Meta title='Basic elements'/>
+
+# Colors
+
+<ColorPalette>
```

Comment:
Is our previous color/palette story still relevant? Should we add a link to it here?